Training24/Header.h

```diff
@@ -1,13 +1,19 @@
 // ------------------------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
+// Kalaimagal V P
 // ------------------------------------------------------------------------------------------------
 // Header.h
-// Program on main branch.
+// Program on A5 branch.
+// Header.h - The functions for sorting integers and searching for a target value.
 // ------------------------------------------------------------------------------------------------
 #ifndef HEADER_H
 #define HEADER_H
 
-char* GetString ();
+/// <summary>Sorts an array of integers in ascending order using Insertion Sort.</summary>
+void InsertionSort (int arr[], int size);
+
+/// <summary>Performs a binary search for a target value in a sorted array.</summary>

```

Comment:
Please describe the return values (especially ones with special meaning like -1 here)